cmd/maya-apiserver/cstor-operator/spc/handler.go

```diff
@@ -342,3 +426,14 @@ func isAutoProvisioning(spc *apis.StoragePoolClaim) bool {
 func isManualProvisioning(spc *apis.StoragePoolClaim) bool {
 	return spc.Spec.BlockDevices.BlockDeviceList != nil
 }
+
+// isSPCDeletionCandidate return true when deletion timestamp & finalizer is
+// available on spc
+func isSPCDeletetionCandidate(spc *apis.StoragePoolClaim) bool {
+	return spc.ObjectMeta.DeletionTimestamp != nil &&
+		isSPCHasFinalizer(spc, spcv1alpha1.SPCFinalizer)
+}
+
+func isSPCHasFinalizer(spc *apis.StoragePoolClaim, finalizer string) bool {
+	return slice.ContainsString(spc.ObjectMeta.Finalizers, finalizer, nil)
```

Comment:
What is there a difference in using `ContainsString` functionality? 
(https://github.com/openebs/maya/pull/1346/files#diff-fd45d2d2eb5c6030a177dc80125877f4R306)